TypeCobol/Compiler/Scanner/Scanner.cs

```diff
@@ -2213,7 +2276,7 @@ private Token ScanSqlCodeOrExecStatementTextOrExecSqlInclude(int startIndex)
             return new Token(TokenType.ExecStatementText, startIndex, endIndex, tokensLine);
         }
 
-        private Token ScanKeywordOrUserDefinedWord(int startIndex)
+        private Token ScanKeywordOrUserDefinedWord(int startIndex, bool tryProcedureName)
```

Comment:
What `tryProcedureName` does ? As I understand it, it restricts to `SectionParagraphName` the returned token type (or partial cobol word). So the naming is not clear.